src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -55,7 +55,7 @@ contains the following details:
 |:--------------------------------------|:---------------------------------------------------------------------------------------------------|
 | `--kubeconfig` `<string>`             | Absolute path(s) to the kubeconfig file(s) (default `$HOME/.kube/config`)
 | `--kubecontext` `<string>`            | kubeconfig context to use
-| `--repository` `<string>`             | The repository from where the various Submariner images will be sourced (default "quay.io/submariner")
+| `--repository` `<string>`             | The repository from where the various Submariner images will be sourced (default "quay.io/submariner", example: "quay.io/<username>" the image in the alternate repo needs to be tagged with "devel")
```

Comment:
\<username\> needs to use escape sequence for proper rendering.

Also, AFAICT its not mandatory to have the image tagged as "devel". The requirement is that when "image:version" is specified, version should not be empty.

Reply:
Thanks for taking a look @sridhargaddam. Removed the brackets. The main reason I added the doc change was because it appends the tag to the repo namae, e.g. `quay.io/networkstatic:test/submariner-operator:devel`

For example:

```sh
$ subctl deploy-broker --repository=quay.io/networkstatic:test  --globalnet

# pod log
Error from server (BadRequest): container "submariner-operator" in pod "submariner-operator-7cb57ff9-trpst" is waiting to start: InvalidImageName

# pod description

  submariner-operator:
    Container ID:
    Image:         quay.io/networkstatic:test/submariner-operator:devel
```

Happy to open a bug and patch it if this is a bug (which after looking closer it looks like one lol). Just let me know. Ty.

Reply:
@sridhargaddam ok, after digging into the code I saw `--version` which will take care of the image tagging. 

So I updated the PR to just add the default version/image tag to the docs with `Image version (default image tag "devel")` and left the description for `--repository` alone.

Thanks!